internal/server/httpServer.go

```diff
@@ -5,62 +5,76 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"hash/crc32"
 	"log/slog"
 	"net/http"
+	"strings"
 	"sync"
 	"time"
 
-	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/internal/comm"
+
 	"github.com/dicedb/dice/internal/clientio"
-	"github.com/dicedb/dice/internal/ops"
-	"github.com/dicedb/dice/internal/querywatcher"
+	"github.com/dicedb/dice/internal/cmd"
 	"github.com/dicedb/dice/internal/server/utils"
+
+	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/internal/ops"
 	"github.com/dicedb/dice/internal/shard"
-	dstore "github.com/dicedb/dice/internal/store"
 )
 
 const Abort = "ABORT"
 
-var unimplementedCommands map[string]bool = map[string]bool{
-	"QWATCH":    true,
-	"QUNWATCH":  true,
-	"SUBSCRIBE": true,
-	Abort:       false,
+var unimplementedCommands = map[string]bool{
+	"QUNWATCH": true,
```

Comment:
We need to think about what QUNWATCH would mean in terms of HTTP.
I think how we can handle it, is in the sync response, we also include ClientIdentifier.

This way the client can send this identifier with Unwatch and we can handle it. @lucifercr07 let me know what you think.

Reply:
Yes sounds good.